Lib/test/test_pulldom.py

```diff
@@ -118,14 +118,15 @@ def test_expandItem(self):
             if evt == pulldom.START_ELEMENT:
                 break
         self.assertEqual("hr", node.tagName,
-            "expandNode did not leave DOMEventStream in the correct state.")
+                         "expandNode did not leave DOMEventStream"
```

Comment:
As below, this line split is also unnecessary. It's best to making such changes.

(I guess autopep8 is being over-zealous.)

Reply:
Don’t change the code when it’s no my a requirement. Keep it as simple as possible. Thanks

Reply:
This probably meant "Don't change the code when it's not a requirement", i.e. avoid unnecessary changes and try to keep them minimal for what you want to achieve.